domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -77,6 +79,33 @@ object InteractionObjectTestBuilder {
     ).build()
   }
 
+  /** Creates fraction part for [NumberWithUnits]*/
+  fun createNumberWithUnitsForFraction(number: Fraction, units: List<NumberUnit>):
+    InteractionObject {
+      val numberWithUnits = NumberWithUnits.newBuilder()
+        .addAllUnit(units)
+        .setFraction(number)
+        .build()
+
+      return InteractionObject.newBuilder().setNumberWithUnits(numberWithUnits).build()
+    }
+
+  /** Creates real part for [NumberWithUnits]*/
+  fun createNumberWithUnitsForReal(number: Double, units: List<NumberUnit>):
+    InteractionObject {
+      val numberWithUnits = NumberWithUnits.newBuilder()
+        .addAllUnit(units)
+        .setReal(number)
+        .build()
+
+      return InteractionObject.newBuilder().setNumberWithUnits(numberWithUnits).build()
+    }
+
+  /** Creates Number unit for [NumberWithUnits]*/
```

Comment:
Suggestion: Creates [NumberUnit] using the [unit] and [exponent] for [NumberWithUnits]. Also please put a full stop at the end everywhere.

Reply:
Done